python_modules/libraries/dagster-snowflake/dagster_snowflake_tests/test_resources.py

```diff
@@ -240,53 +239,3 @@ def test_pydantic_snowflake_resource_duplicate_auth():
             warehouse="TINY_WAREHOUSE",
             private_key="TESTKEY",
         )
-
-
-@mock.patch("snowflake.connector.connect", new_callable=create_mock_connector)
-def test_snowflake_resource_missing_private_key_password(snowflake_connect):
```

Comment:
these tests basically just checked that the `load_pem_private_key` method failed when it wasn't given a password parameter. Now that we always pass a password parameter, these aren't necessary